extensions/3DTILES_binary_buffers/README.md

```diff
@@ -0,0 +1,164 @@
+# 3DTILES_binary_buffers
+
+## Overview
+
+This extension to 3D Tiles enables storage of binary data in external buffers.
+
+## Properties Reference
+
+---------------------------------------
+### 3DTILES_binary_buffers.bufferViews
+
+`bufferViews` provide a typed view into a `buffer`.
+
+**Properties**
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**buffer**|`integer`|The index of the buffer.|☑️ Yes|
+|**byteOffset**|`integer`|The offset relative to the start of the buffer in bytes.|☑️ Yes|
+|**byteLength**|`integer`|The length of the bufferView in bytes.| ☑️ Yes|
+|**elementCount**|`integer`|The number of elements in the buffer view.| ☑️ Yes|
+|**elementByteOffsetsBufferView**|`integer`|The index of the bufferView containing byte offsets for each element. Must be defined for the STRING element type.|No|
```

Comment:
Should there also be an element type for arbitrary sized chunks of binary data? That would look similar to `STRING` and would also use `elementByteOffsetsBufferView`.